recipes/s100py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "s100py" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   url: https://github.com/noaa-ocs-s100/s100py/archive/refs/tags/v{{ version }}.tar.gz
+   sha256: 727b87b4d61d9cbe41249862ac1902adab4db3162b6f7c2c9810439697f70d73
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - h5py
+    - numpy
+    - gdal >=3.2
+
+test:
```

Comment:
You might want to include a `pip check`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

Reply:
Sorry for letting this fall off of my radar. It's probably helpful to add the `pip check`, but if you just want me to merge this, I'm fine with that too.

Reply:
Thanks, let's go ahead with the merge.